packages/storybook-addon/package.json

```diff
@@ -40,6 +40,7 @@
     "*.d.ts"
   ],
   "scripts": {
+    "dev:": "echo noop",
```

Comment:
```suggestion
    "dev": "echo noop",
```

Reply:
Or was this intentional? (same q for the other noop package scripts)

Reply:
no copy pasta